djangocms_versioning_filer/static/filer/js/addons/table-dropzone.js

```diff
@@ -142,45 +127,13 @@ MicroModal.init({
                         uploadWelcome.addClass(hiddenClass);
                         cancelUpload.removeClass(hiddenClass);
 
-                        var fileProceed = true;
+                        let fileProceed = true;
```

Comment:
```suggestion
                        var fileProceed = true;
```

This file is not yet ES6 so it should be ES5